sycl/test-e2e/USM/memadvise_cuda.cpp

```diff
@@ -32,6 +32,7 @@ int main() {
     return -1;
   }
 
+  // NOTE: PI_MEM_ADVICE_CUDA_* advice values are mapped to the HIP backend too.

```

Comment:
Could we maybe create aliases for them and potentially come up with better names for common variants?

Reply:
Open to discuss either. Ideally the flags that UR provides are what the PI ones should may be called. Do you suggest we keep all the `PI_MEM_ADVICE_CUDA_` values and create aliases for hip as well as ones better named that are common? 

Ideally, I don't like the whole _CUDA or _HIP thing in PI because UR will be replacing them whenever that happens with the common named UR flags. However, I feel like for PI, for now it may be best to just create the HIP aliases and leave the common flag naming to when UR completely replaces PI.

Reply:
I agree. Eventually we want something way more common, despite the fact that some of these are backend-specific. Aliases with _HIP is fine for now though.

Reply:
Done.